docs/CNS.md

```diff
@@ -3,100 +3,123 @@
 The computational engine for most of HADDOCK3's modules is the [Crystallography and NMR System (CNS)](http://cns-online.org/v1.3/).
 Once you have [cloned the HADDOCK3 repository](INSTALL.md), please follow these instructions to compile a CNS executable that can be used to run HADDOCK.
 
-## 1 Download CNS
+Before starting, make sure you download the system requirements, they are `gcc`, `gfortran` and `csh`. Luckily they are easily available from the package manager, so if you are using Ubuntu (for example):
 
-Downloading CNS requires a license, which academic users can [request for free](http://cns-online.org) (click on the `Download` menu and fill the form). You will be sent a password for the download via e-mail.
-Using the link provided in the same e-mail, download CNS ("Source installer (compilation required on your system)") to the directory in which you want to install it (for example `software`). Navigate to this directory, then uncompress and extract the archive:
+```bash
+sudo apt-get install gcc gfortran csh
+```
+
+## 1. Downloading CNS
+
+Downloading CNS requires a license, which academic users can [request for free](http://cns-online.org).
+
+To Download CNS, go to their [website](http://cns-online.org) and  click on the `Download` menu and fill the form. You will be sent a password for the download via e-mail - keep in mind only education e-mails are allowed for this.
+
+You will receive a login/password in the e-mail, once you have those go to the [download page](http://cns-online.org/download/v1.3/cns_solve_1.3_all.tar.gz) and use the provided login/password to download the CNS package.
```

Comment:
While here the link is directly for the downloading of the 1.3 version, on the email we get, the link is pointing to [http://cns-online.org/download/](http://cns-online.org/download/).
This page contains multiple download links.
But it is not mentioned that we should download the v1.3.
![Screenshot 2024-05-27 at 10-41-57 Crystallography   NMR System](https://github.com/haddocking/haddock3/assets/132575181/9115fb29-aa10-429e-8717-36abb939022a)